src/components/tag/Tag.root.tsx

```diff
@@ -0,0 +1,63 @@
+import styled, { css } from 'styled-components';
+import { FlattenSimpleInterpolation, ThemedStyledProps } from 'styled-components/ts3.6';
+
+import { Spacing } from '../../foundations/layout';
+import { applyPadding } from '../../foundations/layout/Grid.mixins';
+import { applyShape } from '../../foundations/shape/Shape.mixins';
+import { TTheme } from '../../foundations/theme-provider/themes/theme.types';
+import { applyTextMargin, applyTextStyles, applyTextColor } from '../text/Text.mixins';
+import Text from '../text';
+import { Utils } from '../../shared';
+
+import { PTagRoot } from './Tag.props';
+import { TTagVariant } from './Tag.types';
+
+const TagRoot = styled(Text).withConfig({
+    shouldForwardProp: Utils.forwardProperties(),
+})<PTagRoot>(
+    ({ variant, size, theme }: ThemedStyledProps<PTagRoot, TTheme>): FlattenSimpleInterpolation => {
+        const isMention = variant === 'highlight';
```

Comment:
this can be removed

Reply:
we do use it for three other places so its easier to keep it for clarity IMO